storage/flush_test.go

```diff
@@ -0,0 +1,26 @@
+package storage
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestFlush(t *testing.T) {
```

Comment:
Fine to leave this here for now, but this looks like an integration test rather than a unit test (which typically tests local functionalities than performing an end-to-end test).

Reply:
I put it here to access internal structs to force a flush.